source/evidencevariable/structuredefinition-EvidenceVariable.xml

```diff
@@ -568,5 +568,207 @@
 				<targetProfile value="http://hl7.org/fhir/StructureDefinition/Group"/>
 			</type>
 		</element>
+		<element id="EvidenceVariable.conditional">
+			<path value="EvidenceVariable.conditional"/>
+			<short value="Context in which the variable is observed"/>
+			<definition value="The context, situation, or parameters within which the variable is observed."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Expression"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.classifier">
+			<path value="EvidenceVariable.classifier"/>
+			<short value="Classification"/>
+			<definition value="Classification of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```

Comment:
The ticket indicates classifier should have an Example binding?

Reply:
Added the binding.